internal/provider/resource_issue_alert.go

```diff
@@ -131,7 +131,7 @@ Please note the following changes since v0.12.0:
 				MarkdownDescription: "The issue alert name.",
 				Required:            true,
 				Validators: []validator.String{
-					stringvalidator.LengthBetween(1, 64),
+					stringvalidator.LengthBetween(1, 255),
```

Comment:
Should this be 256?

```suggestion
					stringvalidator.LengthBetween(1, 256),
```

Reply:
I can verify the exact boundary condition, it does seem like according to the error message 256 would be right

Reply:
Ah, I see you merge it but I just verified via API that indeed `256` length is accepted, `257` is rejected. So adding 1 would be more correct (but this is still a lot better than before!)